cmd/exchaind/iaviewer.go

```diff
@@ -0,0 +1,575 @@
+package main
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint"
+	supplytypes "github.com/cosmos/cosmos-sdk/x/supply"
+	evmtypes "github.com/okex/exchain/x/evm/types"
+	slashingtypes "github.com/okex/exchain/x/slashing"
+	tokentypes "github.com/okex/exchain/x/token/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/tendermint/iavl"
+	dbm "github.com/tendermint/tm-db"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"sync"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	acctypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	distypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/okex/exchain/x/distribution/types"
+)
+
+const (
+	KeyDistribution = "s/k:distribution/"
```

Comment:
these keys can be replaced by ModuleBasics in app.go

Reply:
done